packages/lesswrong/lib/utils/viewUtils.ts

```diff
@@ -106,12 +106,6 @@ function getParameters<N extends CollectionNameString>(
     parameters.options.sort = { createdAt: -1 } as any;
   }
 
-  // extend sort to sort posts by _id to break ties, unless there's already an id sort
-  // NOTE: always do this last to avoid overriding another sort
-  if (!(parameters.options.sort && typeof parameters.options.sort._id !== undefined)) {
-    parameters = merge(parameters, { options: { sort: { _id: -1 } } });
-  }
```

Comment:
What's the reason for this change?

Reply:
We upgraded the version of `@typescript-eslint/eslint-plugin`, in order to get access to the `allowTypeImports` option in `.eslintrc.js`. That caused this block to generate a (correct) lint warning, saying that this expression would never be true because `typeof foo` is always a string, but it's compared to `undefined`. We (oetherington and I) tracked this block through history back to Vulcan, and determined that since it's currently broken in a way where it never runs, and that isn't causing any problems, it would be safer to delete it than to change it in a way where it starts doing something.